tcs-persistence/src/main/java/com/transformuk/hee/tis/tcs/service/model/ConditionsOfJoining.java

```diff
@@ -0,0 +1,26 @@
+package com.transformuk.hee.tis.tcs.service.model;
+
+import com.transformuk.hee.tis.tcs.api.enumeration.GoldGuideVersion;
+import java.time.Instant;
+import java.util.UUID;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Id;
+import lombok.Data;
+import org.hibernate.annotations.Type;
+
+/**
+ * A persistable representation of a Conditions of Joining.
+ */
+@Data
+@Entity
+public class ConditionsOfJoining {
+
+  @Id
+  @Type(type = "uuid-char")
```

Comment:
This might be a bit easier on the eyes but is there a reason to have this be different from https://github.com/Health-Education-England/TIS-TCS/blob/ee142daef286ddc2945c5afb7482d8ed1b7090fa/tcs-persistence/src/main/java/com/transformuk/hee/tis/tcs/service/model/ProgrammeMembership.java#L41?

Reply:
None at all, bit of a "this works, `ctrl+c` `ctrl+v`". I'll change it for consistency.